checkov/common/bridgecrew/integration_features/features/custom_policies_integration.py

```diff
@@ -72,5 +81,25 @@ def _get_policies_from_platform(self):
         policies = [p for p in policies if p['isCustom']]
         return policies
 
+    def post_runner(self, scan_reports):
+        if self.bc_cloned_checks.keys():
+            scan_reports.failed_checks = self.extend_records_with_cloned_policies(scan_reports.failed_checks)
+            scan_reports.passed_checks = self.extend_records_with_cloned_policies(scan_reports.passed_checks)
+            scan_reports.skipped_checks = self.extend_records_with_cloned_policies(scan_reports.skipped_checks)
+
+    def extend_records_with_cloned_policies(self, records):
+        new_records = deepcopy(records)
+        for record in records:
+            cloned_policies = self.bc_cloned_checks.get(record.bc_check_id, [])
+            for cloned_policy in cloned_policies:
+                new_record = deepcopy(record)
+                new_record.check_id = cloned_policy['id']
+                new_record.bc_check_id = cloned_policy['id']
+                new_record.guideline = cloned_policy['guideline']
+                new_record.severity = cloned_policy['severity']
+                new_record.check_name = cloned_policy['title']
+                new_records.append(new_record)
+        return new_records
```

Comment:
```suggestion
        bc_check_ids = [record.bc_check_id for records in records]
        for idx, bc_check_id in enumerate(bc_check_ids):
            cloned_policies = self.bc_cloned_checks.get(bc_check_id, [])
            for cloned_policy in cloned_policies:
                new_record = deepcopy(records[idx])
                new_record.check_id = cloned_policy['id']
                new_record.bc_check_id = cloned_policy['id']
                new_record.guideline = cloned_policy['guideline']
                new_record.severity = cloned_policy['severity']
                new_record.check_name = cloned_policy['title']
                records.append(new_record)
        return records
```

how about something like this?